README.md

```diff
@@ -173,6 +173,30 @@ are detailed below:
   If you are an author of an original template not affiliated with any
   organization, only the standard package naming guidelines apply to you.
 
+- **Description:** Write simple, easily-understandable and succinct
+	descriptions.
+	- Keep it short. Try to maximise the content:length ratio and weigh your words
+		thoughtfully. No more than 100 characters, preferably much less.
+	- Avoid the word ‘Typst’, which is redundant unless your package or template
```

Comment:
Can you change these quotes to use "standard" double quotes (`"`) for more consistency with the rest of the README please?